### PR TITLE
docs: update ebpf/k8s example

### PR DESCRIPTION
**Pull request by korniltsev**
Update ebpf/kubernetes example to include kubernetes manifests that can be applied for testing
Add readme

Most of the changes are copy pasted from java/kubernetes example and adopted

### CODE REVIEW

File: examples/grafana-agent-auto-instrumentation/ebpf/kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Grafana Alloy eBPF profiling via auto-instrumentation in Kubernetes
+
+This repository provides a practical demonstration of leveraging Grafana Alloy for continuous application profiling 
+using eBPF and Pyroscope in Kubernetes. It illustrates a seamless approach to profiling Golang and Python processes, 
+aiding in performance optimization.
+
+## Overview
+
+EBPF profiling via Grafana Alloy is based on a few components:
```
Comment (marcsanmi) [inline examples/grafana-agent-auto-instrumentation/ebpf/kubernetes/README.md:9]: Maybe?
```suggestion
eBPF profiling via Grafana Alloy is based on a few components:
```

File: examples/grafana-agent-auto-instrumentation/ebpf/kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- `discovery.kubernetes` for discovering Kubernetes pods
+- `discovery.relabel` for detecting and filtering target processes and setting up labels
+- `pyroscope.ebpf` for enabling EBPF profiling for specific applications
```
Comment (marcsanmi) [inline examples/grafana-agent-auto-instrumentation/ebpf/kubernetes/README.md:12]:
```suggestion
- `pyroscope.ebpf` for enabling eBPF profiling for specific applications
```

**Review by marcsanmi — APPROVED** [review]
LGTM 👍

File: examples/grafana-agent-auto-instrumentation/ebpf/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Overview
+
+eBPF profiling via Grafana Alloy is based on a few components:
+- `discovery.kubernetes` for discovering Kubernetes pods
+- `discovery.relabel` for detecting and filtering target processes and setting up labels
+- `pyroscope.ebpf` for enabling eBPF profiling for specific applications
+- `pyroscope.write` for writing the profiles data to a remote endpoint
+
+Refer to the [official documentation](https://grafana.com/docs/agent/latest/flow/reference/components/pyroscope.ebpf/) for an in-depth understanding and additional configuration options for eBPF with Grafana Alloy.
```
Comment (knylander-grafana) [inline examples/grafana-agent-auto-instrumentation/ebpf/kubernetes/README.md:15]:
```suggestion
Refer to the [official documentation](https://grafana.com/docs/alloy/latest/reference/components/pyroscope/pyroscope.ebpf/) for an in-depth understanding and additional configuration options for eBPF with Grafana Alloy.
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the example and the docs! I have two suggestions to change links from Agent to Alloy documentation.

**Pull request merged**